blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -0,0 +1,119 @@
+---
+title: 3mdeb open-source contributions
```

Comment:
- We need a better title. This title will give a terrible SEO rank.
- What title would the next post about the 3mdeb contribution be? The same?

Reply:
@pietrushnic I changed title. Next title will be iteration of this title.